### DISCUSSION TITLE
Tag ID in the HTTP notification paylod

### DISCUSSION THREAD

**Discussion by ybloch** [AxisCommunications/discussions#39] [VAPIX]
Hello,
Is there a way to receive tag ID in the RFID-tag-reader notifications?

I use a camera intercom of the type:
`AXIS A8207-VE Mk II Network Video Door Station`

**Comment by mattias-kindborg-at-work** [comment]
Hi @ybloch!

Could you please describe what part of VAPIX this would entail, and possibly a curl command or some kind of code snippet that is showing us what you intend to accomplish?

**Reply by ybloch** [reply]
Hi @mattias-kindborg-at-work
The truth is that this question may not be completely related to VAPIX,
I have another question that is related here:
https://github.com/orgs/AxisCommunications/discussions/23#discussioncomment-3317795
if you could help...

What I'm trying to achieve is a notification to my server in some protocol (WS, MQTT, HTTP) when an RFID tag is read with a payload of the ID so that I can manage the whitelist/blacklist on my server.

I saw that there is an option in the AXIS Intercom to configure an address of a server that will receive notifications and to configure an event that will send an HTTP notification to the server when there is an RFID tag detected event.
So I activated it and I receive the events successfully,
what I'm missing is a way to get the ID of the TAG in the event payload

**Reply by mattias-kindborg-at-work** [reply]
Please provide me with a fully working sample where the problem is demonstrated. Preferably create a new GitHub repository with complete code and instructions on how to run the code, and we can work from there. It's hard for us to help you if you don't provide adequate context.
